ledger/catchupaccessor.go

```diff
@@ -62,23 +109,23 @@ const (
 )
 
 // MakeCatchpointCatchupAccessor creates a CatchpointCatchupAccessor given a ledger
-func MakeCatchpointCatchupAccessor(ledger *Ledger, log logging.Logger) *CatchpointCatchupAccessor {
+func MakeCatchpointCatchupAccessor(ledger *Ledger, log logging.Logger) CatchpointCatchupAccessor {
```

Comment:
Why is the output type no longer a pointer if it still returns the address of the CCA?

Reply:
An interface in go is a pointer-type. Hence, there is no need to return a pointer to it.

Reply:
Ah, good to know!